config.json

```diff
@@ -30,6 +30,12 @@
       "values": [],
       "defaultValue": ""
     },
+    "__BlobNamePrefix": {
```

Comment:
Would you do the same for the "container" arg as well? I think it makes sense to have behavior be the same for both the Container arg & the Blob Prefix arg